CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [4.0.0-ga] - 2020-05-25
++ Compatibility Changes
+  + Move config of encryption to under security config, i.e. [encryption] becomes [security.encryption] in tikv config. [#7800](https://github.com/tikv/tikv/pull/7800)
```

Comment:
Where is the release note from? It looks like  duplicated with 4.0.0-rc.2,  but not shown in  our PM's document